src/py/aspen/app.py

```diff
@@ -85,3 +90,19 @@ def logout():
         "client_id": auth0_config.AUTH0_CLIENT_ID,
     }
     return redirect(auth0.api_base_url + "/v2/logout?" + urlencode(params))
+
+
+@requires_auth
+@application.route("/api/usergroup", methods=["GET"])
+def usergroup():
+    with session_scope(db_interface) as db_session:
+        # since authentication is required to access view this information is guaranteed to be in session
+        profile = session["profile"]
+        user = (
+            db_session.query(User)
+            .options(joinedload(User.group))
+            .filter(User.auth0_user_id == profile["user_id"])
+            .one_or_none()
```

Comment:
why is this `one_or_none()`?

Reply:
good point, it will always be one otherwise this would not work at all

Reply:
fixed